src/constants/earthEngine.js

```diff
@@ -262,7 +264,8 @@ export const earthEngineLayers = () => [
         sourceUrl:
             'https://explorer.earthengine.google.com/#detail/MODIS%2FMOD11A2',
         img: 'images/temperature.png',
-        defaultAggregation: ['min', 'max', 'mean'],
+        aggregations: ['min', 'max', 'mean', 'median', 'stdDev', 'variance'],
```

Comment:
Might be useful to have all these strings as constants somewhere in the app?  (an idea for future refactoring)